dist/index.html

```diff
@@ -5,10 +5,10 @@
     <title>Webpack App</title>
     <script>window.defaultRoute = "/patient";</script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
-<link href="/reactapp/css/styles.css" rel="stylesheet"></head>
+</head>
 <body class="cbioOnePageApp">
     <span id="reactHeader"></span>
     <div id="clinical_div"></div>
     <div id="reactRoot"></div>
-<script type="text/javascript" src="/reactapp/js/main.app.js"></script></body>
+<script type="text/javascript" src="http://localhost:3000/reactapp/js/main.app.js"></script></body>
```

Comment:
why is this necessary? Windows thing? I think we should try not to add ports in the source file, since you should be able to override it with env variables on build. Don't feel particularly strongly about it tho, since this is just a dev file.